vms/platformvm/warp/signature.go

```diff
@@ -164,3 +166,7 @@ func VerifyWeight(
 	}
 	return nil
 }
+
+func (s BitSetSignature) String() string {
```

Comment:
Also super minor nit - can we define `String()` above `VerifyWeight` so that we can keep the functions defined on `BitSetSignature` grouped together?